<launch>
<node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager"/>
    <!-- camera node -->
    
  <include file="$(find usb_cam)/launch/usb_cam-test.launch">
 
  <!-- <include file="$(find realsense2_camera)/launch/rs_camera.launch"> -->
  </include>
    <!-- detect aruco markers -->
    <node pkg="nodelet" clear_params="true" type="nodelet" name="aruco_detect" args="load aruco_pose/aruco_detect nodelet_manager">
        <remap from="image_raw" to="/usb_cam/image_raw"/>
        <remap from="camera_info" to="/usb_cam/camera_info"/>
        <param name="length" value="0.7"/>
    </node>

   <!-- <node name = "cupDetectWithFunction" pkg = "cupDetection" type="cupDetectWithFunction" output = "screen"> -->

      <!-- <remap from ="image_raw" to="usb_cam/image_raw"/>   -->
      
   <!-- </node>	 -->
   <node name = "CameraResult" pkg="aruco_pose" type = "CameraResult" output = "screen">
    <!--remap from = "pub_status" to="/master/pub_status"/-->
     <param name=" testStart" type = "int" value = "0"/>
     <param name = "angleMargin" type = "int" value = "30"/>
   </node>
    -->
    <!--
    <node name = "gui" pkg="eurobot_gui"  type="demo_guiForCamera.py"/>
	-->
        <param  name="XPose" type="int" value="490"/>
        <param  name="YPose" type="int" value="315"/>
        <param name = "cupDistance" type = "int" value = "15"/>
</launch>
